custom-connectors/inQubaJourney/apiDefinition.swagger.json

```diff
@@ -0,0 +1,301 @@
+          {
+            "name": "username",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "description": "Account Username",
+            "x-ms-summary": "Username"
+          },
+          {
+            "name": "password",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "description": "Account Password",
+            "x-ms-summary": "Password"
+          },
+          {
+            "name": "grant_type",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "default": "password",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "client_id",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "description": "Client ID",
+            "x-ms-summary": "Client ID"
+          },
+          {
+            "name": "scope",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "internal",
+            "default": "openid cx_api"
+          },
+          {
+            "name": "client_secret",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "description": "Client Secret",
+            "x-ms-summary": "Client Secret"
+          }
+        ]
+      }
+    },
+    "/cems/api/Events": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Create Event",
+        "description": "Publish an event record to the inQuba API. A bearer token is required to interact with the API.",
+        "operationId": "PublishEvent",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "description": "Request Content Type",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "TenantName",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Name of Tenant",
+            "x-ms-summary": "Tenant Name"
+          },
+          {
+            "name": "AuthorizationToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Authorization Token",
+            "x-ms-summary": "Authorization Token"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "eventDefinitionCode": {
+                  "type": "string",
+                  "description": "eventDefinitionCode",

```

Comment:
@prinayinq **All the description values in the swagger** are expected to be free from grammatical and spell errors. Please provide valid description values for all the properties.

Reply:
Updated as required